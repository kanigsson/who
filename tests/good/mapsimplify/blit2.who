logic !! ['a7|r|'e] : ref(r,'a7) -> <r  'e > -> 'a7
goal blit_correct : forall [|r2|]. 
  forall (ar2:ref(r2,prop)).  
  forall (s2:<r2  >).  
    !! [prop| r2| { }] ar2 s2
