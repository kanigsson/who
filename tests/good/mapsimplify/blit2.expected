goal blit_correct : forall (r2:prop). r2
